.github/actions/setup-rust-runtime/action.yaml

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think we can probably skip this as they are basically optimizations for the much larger datafusion codebase.

Reply:
@alamb Thanks for your corrections. All comments are resolved now.